backend/pkg/api/types/validator_dashboard.go

```diff
@@ -269,6 +274,59 @@ type VDBTotalWithdrawalsData struct {
 
 type InternalGetValidatorDashboardTotalWithdrawalsResponse ApiDataResponse[VDBTotalWithdrawalsData]
 
+// ------------------------------------------------------------
+// Rocket Pool Tab
+type VDBRocketPoolTableRow struct {
+	Node      Address         `json:"node"`
+	StakedEth decimal.Decimal `json:"staked_eth"`
+	Minipools struct {
+		Total uint64 `json:"total"`
+		Leb16 uint64 `json:"leb_16"`
+		Leb8  uint64 `json:"leb_8"`
+	} `json:"minipools"`
+	Collateral    PercentageDetails[decimal.Decimal] `json:"collateral"`
+	AvgCommission float64                            `json:"avg_commission"`
+	Rpl           struct {
+		Claimed   decimal.Decimal `json:"claimed"`
+		Unclaimed decimal.Decimal `json:"unclaimed"`
+	} `json:"rpl"`
+	EffectiveRpl  decimal.Decimal `json:"effective_rpl"`
+	RplApr        float64         `json:"rpl_apr"`
+	SmoothingPool struct {
+		OptIn     bool            `json:"opt_in"`
+		Claimed   decimal.Decimal `json:"claimed"`
+		Unclaimed decimal.Decimal `json:"unclaimed"`
+	} `json:"smoothing_pool"`
+}
+type InternalGetValidatorDashboardRocketPoolResponse ApiPagingResponse[VDBRocketPoolTableRow]
+
+type InternalGetValidatorDashboardTotalRocketPoolResponse ApiDataResponse[VDBRocketPoolTableRow]
+
+type VDBNodeRocketPoolData struct {
+	Timezone      string          `json:"timezone"`
+	RefundBalance decimal.Decimal `json:"refund_balance"`
+	DepositCredit decimal.Decimal `json:"deposit_credit"`
+	Penalties     uint64          `json:"penalties"`
+	RplStake      struct {
+		Min decimal.Decimal `json:"min"`
+		Max decimal.Decimal `json:"max"`
+	} `json:"rpl_stake"`
+}
+
+type InternalGetValidatorDashboardNodeRocketPoolResponse ApiDataResponse[VDBNodeRocketPoolData]
+
+type VDBRocketPoolMinipoolsTableRow struct {
+	Node            Address         `json:"node"`
+	ValidatorIndex  uint64          `json:"validator_index"`
+	MinipoolStatus  string          `json:"minipool_status" tstype:"'initialized' | 'prelaunch' | 'staking' | 'withdrawable' | 'dissolved'" faker:"oneof: initialized, prelaunch, staking, withdrawable, dissolved"`
+	ValidatorStatus string          `json:"validator_status" tstype:"'active' | 'withdrawn' | 'withdrawal_requested' | 'withdrawal_processing' | 'withdrawal_processed' | 'withdrawal_failed'" faker:"oneof: active, withdrawn, withdrawal_requested, withdrawal_processing, withdrawal_processed, withdrawal_failed"`
```

Comment:
I believe that the possible validator statuses will be these https://github.com/gobitfly/beaconchain/pull/576/files#diff-be6ea31c064ada3fca256c81402e190ae2c1556fc439d927bc35afe2aec96405R22-R31 . This still has to be implemented in the summary validators call.